open-hackathon-server/src/hackathon/hk8s/k8s_service_adapter.py

```diff
@@ -43,39 +43,58 @@ def __init__(self):
         #self.k8s_client = client.ApiClient()
         return
 
-    def create_k8s_depolyment_with_yaml(self, yaml):
-        k8s_api = utils.create_from_yaml(self.k8s_client, "nginx-deployment.yaml")
-        deps = k8s_api.read_namespaced_deployment("nginx-test", "default")
-        #self.log.debug("Deployment {0} created".format(deps.metadata.name))
-        return deps.metadata.name
+    def create_k8s_deployment_with_yaml(self, yaml, name, namespace):
+        k8s_api = utils.create_from_yaml(self.k8s_client, yaml)
+        resp = k8s_api.read_namespaced_deployment(name, namespace)
+        return str(resp.status)
```

Comment:
If k8s returns 200, does it mean that the request is **completed** or just **accepted**?

Reply:
not sure so far, how about we let it in for now and debug it when it's ready to run.